src/core/router/transports/ssu/packet.cc

```diff
@@ -217,12 +217,13 @@ std::uint8_t const* SSUSessionCreatedPacket::GetDhY() const {
 void SSUSessionCreatedPacket::SetIPAddress(
     std::uint8_t* address,
     std::size_t size) {
-  m_IPAddress = address;
+  m_IPAddress.reset(new std::uint8_t[size]);
+  memcpy(m_IPAddress.get(), address, size);
```

Comment:
I know what `<cstring>` does, but will you consider `std::memcpy` for consistency?